apps/studio/src/components/quicksearch/QuickSearch.vue

```diff
@@ -146,6 +146,7 @@ export default Vue.extend({
       // /me *evil laugh*
       result['ctrl+p'] = this.selectUp
       result['ctrl+n'] = this.selectDown
+      result['right'] = this.persistentSearchEnter
```

Comment:
how do we signal to users that pressing right will keep the quick search open?

Reply:
Good question, I hadn't really thought about it. VS code doesn't really show this well, so I couldn't really get inspiration from there. 
We could add it to the initial quick search popup (I'll add a picture below of what that could look like), but I don't know if that is the right place to do this.
![Screenshot from 2023-11-30 09-49-28](https://github.com/beekeeper-studio/beekeeper-studio/assets/25231820/3da3f9b5-dcd4-4456-b637-eead38c68562)

Reply:
You could have an entry for each:
- Open in background: [Right Arrow]
- Alt in background: [Meta + Right Arrow]